### DISCUSSION TITLE
Different lock files for different poetry versions?

### DISCUSSION THREAD

**Discussion by igorvoltaic** [python-poetry/poetry#9521] [Q&A] [answered]
I've recently updated poetry 1.7.1 -> 1.8.2 and it says all my lock files significantly different from pyproject. Is that expected behavior?

**Comment by radoering** [comment, marked as answer]
That's #8737. See also the corresponding [section in the blogpost](https://python-poetry.org/blog/announcing-poetry-1.8.0/#upgrading-warning-about-inconsistent-lockfile-to-an-error).

In short, your lock files have already been inconsistent before the update and Poetry 1.7 printed a warning. With Poetry 1.8, this warning is now an error.
